plugins/inputs/docker/docker.go

```diff
@@ -297,6 +316,25 @@ func (d *Docker) gatherContainer(
 		}
 	}
 
+	inspectCtx, inspectCancel := context.WithTimeout(context.Background(), d.Timeout.Duration)
+	defer inspectCancel()
+	ic, err := inspectWrapper(d.client, inspectCtx, container.ID)
+	if err != nil {
+		return err
+	}
+	// Iterate over the container environment variables and see if there are any variables to be tagged
+	for _, env := range ic.Config.Env {
```

Comment:
Nvm, the Envs are a list and not a map so its just a list of [VALUE1=foo, VALUE2=foo2], but I do check if it's in the whitelisted on line 33